packages/js-client/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@aragon/sdk-client-template",
+  "author": "Aragon Association",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "TESTING=true tsdx test --verbose --runInBand",
+    "lint": "tsdx lint",
+    "prepare": "tsdx build",
+    "size": "size-limit",
+    "analyze": "size-limit --why",
```

Comment:
```
➜  js-client git:(f/OS-465-dummy-js-client) yarn analyze
yarn run v1.22.19
$ size-limit --why
 ERROR  Argument --why works only with @size-limit/webpack plugin and @size-limit/webpack-why plugin.
        You can add Bundle Analyzer to you own bundler.
error Command failed with exit code 1.
```

Reply:
@josemarinas

Reply:
This is working but we are waaay over the size limit

![imagen](https://github.com/aragon/osx-plugin-template-hardhat/assets/36479864/f5d621a4-1946-4e9b-a9df-6048d209e2a2)

Reply:
Is there any way to easily reduce what we export? If not, we'll need a separate task to optimize this

Reply:
We can increase it to 1MB maybe. The ethers, osx-ethers and  graphql are pretty big packages so maybe the problem comes from there. Not sure